stdlib/Mmap/src/Mmap.jl

```diff
@@ -155,6 +157,8 @@ privileges are required to grow the file.
 The `shared` keyword argument specifies whether the resulting `Array` and changes made to it
 will be visible to other processes mapping the same file.
 
+The `exec` keyword argument specifies whether the underlying mmap data will be executale.
```

Comment:
```suggestion
The `exec` keyword argument specifies whether the underlying mmap data will be executable.
```

Reply:
I wonder why the typo check didn't catch this. @LilithHafner, any idea?

Reply:
Because typos-ci is a list of corrections, not a list of valid words, and prioritizes reducing false-positives over reducing false-negatives.

Fix: https://github.com/crate-ci/typos/issues/921#issuecomment-1964120104

Reply:
> ```
>commmitment -> commitment
>matcbh -> match
>instancs -> instances
>veriier -> verifier
>absrobed -> absorbed
>```

So someone has to hit your exact misspelling and have it added to the list. I agree that will reduce false positives heh.